xbmc/filesystem/File.h

```diff
@@ -109,8 +110,7 @@ class CFile
   int64_t GetLength();
   void Close();
   int GetChunkSize();
-  std::string GetContentMimeType(void);
-  std::string GetContentCharset(void);
+  const std::string GetProperty(XFILE::FileProperty type, const std::string &name = StringUtils::Empty) const;
```

Comment:
See my other comment re. `StringUtils::Empty`. I would use `""` - also saves you the new dependency to `StringUtils.h`.

Reply:
... and shouldn't you add `override` here?

Reply:
No override, its not derived from anything.

Reply:
Okay, overlooked that CFile is not derived from IFile. :-/